src/components/RightSidebar/RightSidebar.vue

```diff
@@ -375,6 +375,20 @@ export default {
 	flex-direction: column;
 }
 
+// FIXME upstream: move styles to nextcloud-vue library
+:deep(.app-sidebar-tabs__nav) {
+	padding: 0 10px;
+
+	.checkbox-radio-switch__label {
+		text-align: center;
+		justify-content: flex-start;
+	}
+
+	.checkbox-radio-switch__icon {
+		flex-basis: auto;
+	}
+}
```

Comment:
If I'm not wrong, `checkbox-radio-switch__label` and `checkbox-radio-switch__icon` are not in deep.

Reply:
Somehow they work as-is and don't work inside :deep()